npc/quests/quests_13_2.txt

```diff
@@ -5671,8 +5671,7 @@ spl_in01,30,324,3	script	High Laphine#grenouille	446,{
 					set ep13_2_tre1,6;
 					changequest 2185,2186;
 					getitem 6081,3; //Splendide_Coin
-					if (checkre(3) == 0)
-						getexp 0,(checkre(3))?50000:700000; //unconfirmed (RE)
+					//getexp 0,(checkre(3))?70000:700000; //iRO only
```

Comment:
This removes pre-renewal bonus as well?

Reply:
Yeah, it's commented on both aegis renewal an pre-renewal